packages/lab/stories/components/CloseTabWarningDialog.tsx

```diff
@@ -1,10 +1,10 @@
 import { Button } from "@salt-ds/core";
 import {
-  TabDescriptor,
   Dialog,
   DialogTitle,
   DialogContent,
   DialogActions,
+  TabDescriptor,
```

Comment:
Are the changes in this file necessary? They don't look to be restructuring?

Reply:
this CloseTabWarningDialog was used by 2 files where one was wanting the tabs to be string and the other a TabDescriptor, so i merged the types when i moved it to /components